robot_state: '/state'
robot_topopath: '/topopath'
interest_points: '/interest_points'
afference: '/afference'
destinations_log: '/destinations_log'
pose_topic: '/amcl_pose'
goal_topic: '/move_base_simple/goal'
make_plan: '/move_base/NavfnROS/make_plan'

robot_max_speed: 0.26